libraries/standard/json/src/json.c

```diff
@@ -564,18 +599,28 @@ static bool_ skipDigits( const char * buf,
                          size_t * start,
                          size_t max )
 {
-    size_t i = *start, save = *start;
+    bool_ ret = false;
+    size_t i, save;
```

Comment:
`save` what?

Reply:
save the starting index

Reply:
Might be clearer to name it `save_starting_index` then.

Reply:
changed